test/jdk/javax/crypto/EncryptedPrivateKeyInfo/GetAlgName.java

```diff
@@ -61,8 +56,16 @@ public static void main(String[] argv) throws Exception {
             AlgorithmParameters ap = c.getParameters();
             epki = new EncryptedPrivateKeyInfo(ap, BYTES);
             if (!epki.getAlgName().equalsIgnoreCase(algo)) {
-                System.out.println("...expect: " + algo);
-                System.out.println("...got: " + epki.getAlgName());
+                System.out.println("...expected: " + algo);
+                System.out.println("     ...got: " + epki.getAlgName());
+                status = false;
+            }
+
+            AlgorithmParameters ap2 = AlgorithmParameters.getInstance(ap.getAlgorithm());
```

Comment:
It would be useful to add a comment here that you are testing that an EPKI can be created with an uninitialized AP.